src/components/Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../../api";
+
+const Comments = ({ articleId }) => {
+    const [comments, setComments] = useState([])
+    const [loading, setLoading] = useState(true)
+    
+    useEffect(() => {
+        getCommentsByArticleId(articleId)
+          .then((data) => {
+            setComments(data)
+            setLoading(false)
+          })
+      }, [articleId])
+    
+      if (loading) { 
+        return <p style={{fontSize: 40}}>Loading comments...</p>
+      }
```

Comment:
It would be nice if we had something in here to display a message to the user if there aren't any comments associated with the article, rather than just having an empty section.